src/components/attendee/Input/Input.module.scss

```diff
@@ -8,25 +8,17 @@
 .input {
   display: flex;
   flex-direction: column;
+}
 
-  label {
-    width: 100%;
-  }
-
-  label::after {
-    content: attr(data-label);
-    position: absolute;
-    top: space.empty(0.75);
-    display: flex;
-    align-items: center;
-    padding: 0 0 0 space.empty(0.75);
-    height: 26px;
-    color: theme.$c-glass-panel-text;
-    right: space.empty(0.8);
-    border-left: border.$width--slim solid theme.$sparkle--border;
-    color: theme.$c-inverted-text;
-    cursor: pointer;
-  }
+label {
+  position: absolute;
```

Comment:
Shouldn't the flexbox of the Input parent determine how to layout the children like `input` and `label` properly?

Reply:
[NOTE]

Also, I just realized this is not the original label of being just an explanatory text above the input, but rather a subtext label. In some earlier components there was a prop that determined where the label is shown, others had 2 labels taken as props

Reply:
maybe we can rename it?

in props
label -> actionButton? (sideLabel?)

and in the component
use button or span for it?

Reply:
It can be renamed now or later, we should first have a clear picture of what makes an `Input` component. Does design require it, does separate label for any input make more sense, etc.

Reply:
removed and refactored attendee input component a bit. Check out here: https://github.com/sparkletown/sparkle/pull/3005/commits/56622b7f4483b16d4282f57daa294296e318b5dc